lib/logstash/outputs/sqs/patch.rb

```diff
@@ -0,0 +1,18 @@
+# This patch was stolen from logstash-plugins/logstash-output-s3#102.
```

Comment:
FWIW, `logstash-input-sqs` also needs this patch. Perhaps it should be moved to `logstash-mixin-aws`?